File: 2348/solution.py
class Solution:
    def zeroFilledSubarray(self, nums: List[int]) -> int:
        soln = 0
        tmp = 0
        for elem in nums:
            if elem == 0:
                tmp += 1
                soln += tmp
            else:
                tmp = 0
        return soln

File: 2348/Solution.java
class Solution {
    public long zeroFilledSubarray(int[] nums) {
        long soln = 0;
        long tmp = 0;
        for (int elem : nums){
            if (elem == 0){
                tmp = tmp + 1;
                soln = soln + tmp;
            } else {
                tmp = 0;
            }
        }
        return soln;
    }
}